### PR TITLE
Fixes #13910 - Auto populate env, cv on new host

### PR DESCRIPTION
**Pull request by parthaa**
Adds logic to auto populate env, cv on new hosts page when a host group
is selected

### CODE REVIEW

**Comment by parthaa** [comment]
Testing Procedure

- Create a CV, Env
- Go to hostgroups page and create a new host group with env, cv combination
- Go to new hosts page 
- Fill in name, location, hostgroups field. CV and Env should get automatically repopulated by whats selected in the host group.

File: app/helpers/katello/hosts_and_hostgroups_helper.rb [hunk 1]
```diff
@@ -68,8 +71,12 @@ def content_views_for_host(host, options)
         include_blank = '<option></option>'
       end
 
-      content_view = content_view(host)
+      selected_host_group = options.fetch(:selected_host_group, nil)
       lifecycle_environment = lifecycle_environment(host)
+      lifecycle_environment = lifecycle_environment(selected_host_group) if lifecycle_environment.nil? && selected_host_group.present?
```
Comment (ehelms) [inline app/helpers/katello/hosts_and_hostgroups_helper.rb]: I'm sensing some repeated logic between this and https://github.com/Katello/katello/pull/5814/files#diff-a29a1f54f7213ac056cf5af0631d90fcR45

**Comment by johnpmitsch** [comment]
This works as expected when I tested out the new hosts page :+1:

**Comment by johnpmitsch** [comment]
@parthaa looks like this was already a bug http://projects.theforeman.org/issues/12899

**Comment by parthaa** [comment]
@ehelms @johnpmitsch updated

**Comment by parthaa** [comment]
@jlsherrill updated

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**